src/libsyncengine/jobs/network/downloadjob.cpp

```diff
@@ -433,8 +450,9 @@ bool DownloadJob::createLink(const std::string &mimeType, const std::string &dat
         LOGW_DEBUG(_logger, L"Create symlink: " << Utility::formatSyncPath(targetPath).c_str() << L", "
                                                 << Utility::formatSyncPath(_localpath).c_str());
 
+        bool isFolder = mimeType == mimeTypeSymlink ? false : true;
```

Comment:
```suggestion
        bool isFolder = mimeType == mimeTypeSymlinkFolder;
```
Not wrong, but this would be easier to read